### PR TITLE
add project description contact information and keyword

### PR DESCRIPTION
**Pull request by markcsims**
https://asaphub.atlassian.net/browse/GP2-69

### CODE REVIEW

File: packages/gp2-components/src/templates/ProjectOverview.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+const contentStyles = css({
+  marginTop: rem(32),
+});
+
+const columnStyles = css({
+  display: 'grid',
+  columnGap: rem(32),
+  gridRowGap: rem(32),
+  [`@media (min-width: ${smallDesktopScreen.min}px)`]: {
```
Comment (jhonny-m) [inline packages/gp2-components/src/templates/ProjectOverview.tsx:29]: I usually use `crossQuery`

File: apps/gp2-server/src/data-providers/project.data-provider.ts [hunk 2]
```diff
@@ -104,13 +104,21 @@ export function parseProjectToDataObject(
...
-    projectProposalUrl: project.flatData.projectProposal || undefined,
+    id,
+    title: project.title || '',
+    startDate: project.startDate || '',
+    endDate: project.endDate || undefined,
+    status: project.status,
+    projectProposalUrl: project.projectProposal || undefined,
+    pmEmail: project.pmEmail || undefined,
+    leadEmail: project.leadEmail || undefined,
+    description: project.description || undefined,
```
Comment (jhonny-m) [inline apps/gp2-server/src/data-providers/project.data-provider.ts:120]: I've defaulted the description to `''` in working groups and have it as a mandatory string. not sure which is the right approach, we should check and make both the same

**Review by jhonny-m — APPROVED** [review]

File: apps/gp2-server/src/data-providers/project.data-provider.ts (hunk 2 above)
Comment (markcsims) [reply apps/gp2-server/src/data-providers/project.data-provider.ts:120]: on this, it's not mandatory

File: packages/gp2-components/src/templates/ProjectOverview.tsx (hunk 1 above)
Comment (markcsims) [reply packages/gp2-components/src/templates/ProjectOverview.tsx]: done

**Pull request merged**